test_project/settings.py

```diff
@@ -65,11 +68,13 @@
 DATABASES = {
     'default': {
         'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': ':memory:',
-    },
+        'NAME': BASE_DIR / 'db.sqlite3',
+    }
 }
 
+
 DEFAULT_AUTO_FIELD = 'django.db.models.AutoField'
+PRIMARY_KEY_FIELD = 'django.db.models.AutoField'
```

Comment:
The setting name being `PRIMARY_KEY_FIELD` that's used just for the project can be more descriptive. Maybe this gets updated to something like `EAV2_PRIMARY_KEY_FIELD` or `DJANGO_EAV2_PK_TYPE`